### ISSUE TITLE
F1 rules doesn't recognize fields in html attributes

### ISSUE THREAD

**Issue by Kabulan0lak** [looker-open-source/look-at-me-sideways#118]
The F1 rules triggers in this particular set up:

```
  dimension: dim1 {}

  dimension: dim2 {
    html: <span>{{ dim1._rendered_value }}</span>;;
  }
```

In this case, it looks like the `dim1` is seen as a view by LAMS, but it's just a field from the very view we are defining.

**Comment by Kabulan0lak** [comment]
I see! Indeed that should do the trick 🔥

**Comment by fabio-looker** [comment]
Should now be fixed in v2.1.6
